### ISSUE TITLE
[LR] Impossible to upload a file a in manual test case execution

### ISSUE THREAD

**Issue by ittaLaRedoute** [cerberustesting/cerberus-core#2412]
Hello,

When we execute a manual test, we can't upload a file in an action or control. 
An error 500 occured at the endpoint call.

Tried with txt, png, and xml file.

![image](https://user-images.githubusercontent.com/97182913/196987309-a3a8e285-6276-4aba-8651-97aac46e33d5.png)

Cerberus version: 4.15-SNAPSHOT-1740

Lucas HIMPENS
Thanks

**Comment by ittaLaRedoute** [comment]
Hello, the upload for a file works at a step level, but for at a control level, it doesn't work: when clicking on "Save", page reload, but the file isn't uploaded.

**Comment by bcivel** [comment]
Hello,
I can't reproduce locally, neither on QA
https://qa.cerberus-testing.com/TestCaseExecution.jsp?executionId=484803

Could you please provide more detail, or maybe reproduce on QA?
Thanks

**Comment by ittaLaRedoute** [comment]
Hello @bcivel ,
The behaviour has changed since we opened the issue. Now when we click on the upload button of a Control the file is uploaded but at the action level and not at the control level.
![image](https://user-images.githubusercontent.com/97182913/201605817-caede2d5-5166-4a3b-b71e-4f8c3d29b37b.png)
In the screenshot, I uploaded two files:
- The first is a PNG I uploaded with the button on the action
- The second is a txt file I uploaded with the button on the control. But the txt file is placed on the action and not on the control. Is this the expected behaviour ?
